doc/source/user_guide/machine_learning_pysteps.rst

```diff
@@ -0,0 +1,103 @@
+This is a natural attempt to filter out the unpredictable precipitation features, which would otherwise increase the variance (and the MSE).
+The same principle holds for convolutional and/or deep neural network architectures, which also produce smooth nowcasts.
+
+Therefore, you should never directly compare an error-minimization machine learning nowcast to a variance-preserving radar extrapolation, as produced by the module pysteps.nowcasts.extrapolation. Instead, you should use the pysteps ensemble mean.
+
+A pysteps "ensemble mean" can be generated using the modules pysteps.nowcasts.sprog, pysteps.nowcasts.sseps or pysteps.nowcasts.anvil. 
+Another possibility, but more computationally demanding, is to average many ensemble members generated by the modules pysteps.nowcasts.steps or pysteps.nowcasts.sseps. 
+
+Still, even by using the pysteps ensemble mean, it is not given that its variance will be the same as the one of machine learning predictions. 
+Possible solutions are to:
+
+#. use a normalized MSE or other score,
+#. decompose the field with a Fourier (or wavelet) transform to compare features at the same spatial scales.
+
+A good deterministic comparison of a deep convolutional machine learning neural network nowcast and pysteps is given in :cite:`FNPC2020`.
+
+2. Probabilistic machine learning vs pysteps
+--------------------------------------------
+
+Probabilistic machine learning regression methods can be roughly categorized into:
+
+a. Quantile-based methods, such as quantile regression, quantile random forests and quantile neural networks.
+b. Ensemble-based methods, such as generative adversarial neural networks (GANNs) and variational auto-encoders (VAEs).
+
+Quantile-based machine learning nowcasts are interesting, but can only estimate the probability of exceedance at a given point (see e.g. :cite:`FSNBG2019`).
+
+To estimate areal exceedance probabilities, for example above catchments, or to propagate the nowcast uncertainty into hydrological models, the full ensemble still needs to be generated, e.g. with generative machine learning models.
+
+Generative machine learning methods are similar to the pysteps ensemble members. Both are designed to produce an ensemble of possible realizations that preserve the variance of observed radar fields.
+
+A proper probabilistic verification of generative machine learning models against pysteps would be an interesting research direction.
+
+Summary
+-------
+The table below is an attempt to classify machine learning and pysteps nowcasting methods according to the four main prediction types:
+
+#. Deterministic (variance-preserving)
+#. Deterministic (error-minimization)
+#. Probabilistic (quantile-based)
+#. Probabilistic (ensemble-based)
+
+The comparison of methods from different types should only be done carefully and with good reasons.
+
+.. list-table::
```

Comment:
this table renders only with single-line rows... not sure how to fix this, though...

Reply:
I tried to implement this but without success:
https://rackerlabs.github.io/docs-rackspace/tools/rtd-tables.html

All steps were already done, except setting the html_context variable, which did not work.

Reply:
Sorry for the delay. I missed this issue :S 

Commit 72fe8b5 should fix this ;)  This hack also fixes the problems with the of the single-line rows in the Developers documentation.